views/base.tpl

```diff
@@ -345,6 +345,7 @@
                 <a href="/steam/compositor">Toggle Compositor</a>
                 <a href="/mangohud">Toggle MangoHud</a>
                 <a href="/logout">Log Out</a>
+                <a href="/type">Type Something</a>
```

Comment:
I think "Log Out" should be on the bottom.

Also, would "Virtual Keyboard" be a better name?

Reply:
Yeah, that name is much better & done